build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+BUILD_DIR=build-aws
+DIST_DIR=dist-aws
+S3_BUCKET=carbon-deploy
```

Comment:
This same `S3_BUCKET=carbon-deploy ` is used for reading the drivers and writing the output at this time?